### PR TITLE
[openwebnet] Fixed handling of ZigBee USB gateway and devices (fixes #8915 #8917 #8962) (2.5.x)

### LINKED ISSUE #8915: [openwebnet] Config address (where) for Zigbee devices are not accepted
## Current Behavior
Addresses like `where="111222301"` configured via file for OpenWebNet Zigbee devices are not accepted and an
`ERROR IllegalArgumentException: WHERE address is invalid` 
is returned in logs.

## Expected Behavior
Zigbee addresses should be handled correctly and, in case they are wrongly configured, an INFO error should be logged and Thing put offline.

### LINKED ISSUE #8917: [openwebnet] 2-units Zigbee switch modules are not discovered and handled correctly
2-units Zigbee switch modules are not discovered correctly and update messages coming from network are not correctly handled

### PR DESCRIPTION
**Pull request by mvalla**
**BREAKING CHANGE** Configuration for OpenWebNet ZigBee devices has changed, see the binding README for the current configuration of the 'where' parameter for Zigbee (radio) devices and examples. Existing things configuration should be updated.

---
This PR is a backport of main-#9076 (already merged) to 2.5.x.

* [openwebnet] Fixed config, handling and discovery of ZigBee devices (fixes #8915 and fixes #8917). Added test for ownID. Now uses openwebnet4j v0.3.2
* [openwebnet] ZigBee: improved handling of wrong 'where' parameter. Updated README. Improved logging.
* [openwebnet] Improved discoveryByActivation to distinguish dimmers from lights. Now using openwebnet4j 0.3.2-1

### CODE REVIEW

File: bundles/org.openhab.binding.openwebnet/src/main/java/org/openhab/binding/openwebnet/handler/OpenWebNetBridgeHandler.java [hunk 1]
```diff
@@ -131,11 +135,11 @@ public void initialize() {
      * Init a ZigBee gateway based on config
      */
     private @Nullable OpenGateway initZigBeeGateway() {
-        logger.debug("Initializing ZigBee USB gateway");
+        logger.debug("Initializing ZigBee USB Gateway");
         OpenWebNetZigBeeBridgeConfig zbBridgeConfig = getConfigAs(OpenWebNetZigBeeBridgeConfig.class);
         String serialPort = zbBridgeConfig.getSerialPort();
         if (serialPort == null || serialPort.isEmpty()) {
-            logger.warn("Cannot connect to gateway. No serial port has been provided in Bridge configuration.");
+            logger.info("Cannot connect ZigBee USB Gateway. No serial port has been provided in Bridge configuration.");
```
Comment (fwolter) [inline bundles/org.openhab.binding.openwebnet/src/main/java/org/openhab/binding/openwebnet/handler/OpenWebNetBridgeHandler.java:142]: Why did you change this? Logging to info should be used sparsely e.g. a newly started component or a user file that has been loaded. Same for below.

**Review by fwolter — CHANGES_REQUESTED** [review]

File: bundles/org.openhab.binding.openwebnet/src/main/java/org/openhab/binding/openwebnet/handler/OpenWebNetBridgeHandler.java (hunk 1 above)
Comment (mvalla) [reply bundles/org.openhab.binding.openwebnet/src/main/java/org/openhab/binding/openwebnet/handler/OpenWebNetBridgeHandler.java]: done. changed info>warn for config problems

**Review by fwolter — APPROVED** [review]
LGTM Can you file a follow-up PR to fix the logging in the main branch, too?

**Comment by mvalla** [comment]
> LGTM Can you file a follow-up PR to fix the logging in the main branch, too?
Done: #9159

**Comment by mvalla** [comment]
@fwolter can you add here the bug tag? for release notes...

**Comment by fwolter** [comment]
Done
